pkgs/development/python-modules/wtforms/default.nix

```diff
@@ -34,4 +35,14 @@ buildPythonPackage rec {
     maintainers = [ maintainers.bhipple ];
   };
 
+  passthru.wtforms2 = wtforms.overridePythonAttrs (old: rec {
+    version = "2.3.3";
+    src = fetchPypi {
+      inherit (old) pname;
+      inherit version;
+      sha256 = "81195de0ac94fbc8368abbaf9197b88c4f3ffd6c2719b5bf5fc9da744f3d829c";
+    };
+    doCheck = false;
+    propagatedBuildInputs = [ markupsafe ];
+  });
```

Comment:
This is not how we do such things in pythonPackages. I assume the packages share the module name which means we can't do it like this and need to overwrite it for every end application that we want to fix.

Reply:
feel free to suggest an alternative, but I'd rather revert https://github.com/NixOS/nixpkgs/pull/155221 and override it for the single package that needs 3.0 (which hasn't even been merged yet?) rather than copy-paste this into like 10 different packages :(

Reply:
This is is forbidden. We cannot use it as a stop gap. It causes massive issues when the two versions end up in a single python env. We can also revert the update of wtforms and override it for the one package that actually needs it.